plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -350,8 +366,6 @@ public void modifyText(ModifyEvent event) {
    */
   private final class AutoPackageNameSetterOnGroupIdChange implements VerifyListener {
```

Comment:
If you want, you can change it to `ModifyListener` and close #906. That way, you don't have to do the complex computation to get `newGroupId`.